cmd/agent/app/flags_test.go

```diff
@@ -51,7 +57,7 @@ func TestBingFlags(t *testing.T) {
 	assert.Equal(t, 42, cfg.Processors[2].Server.QueueSize)
 	assert.Equal(t, 42, cfg.Processors[2].Workers)
 
-	err = flags.Parse([]string{
+	err = command.ParseFlags([]string{
```

Comment:
:1 I think it can be removed. Before error was caused because of double parsing.